src/app/module-nav/module-nav.demo.ts

```diff
@@ -47,6 +47,13 @@ export class ModuleNavDemoComponent implements AfterViewInit {
 
   onModuleButtonClick(e: any) {
     console.info('Module Nav Role Button clicked', e);
+    this.moduleNavSwitcher?.toggleModuleButtonFocus(true);
+    this.accordion?.deselectAll();
```

Comment:
@krishollenbeck these two methods demonstrate how to control the EP components with the new API methods